src/librustc_mir/borrow_check/nll/region_infer/mod.rs

```diff
@@ -403,10 +422,23 @@ impl<'tcx> RegionInferenceContext<'tcx> {
         infcx: &InferCtxt<'_, 'gcx, 'tcx>,
         mir: &Mir<'tcx>,
         mir_def_id: DefId,
+    ) -> Option<ClosureRegionRequirements<'gcx>> {
+        common::time(infcx.tcx.sess, &format!("solve({:?})", mir_def_id), || {
```

Comment:
Consider using something a little more specific than just "solve" here, like "solve region constraints", just to ease lives of other users of `-Z time-passes`